# Topics
camera_topic: "/axis/image_raw/compressed"
camera_info: "/axis/camera_info"
lidar_topic: "/robot/FC_ouster/points"

# Dynamic rqt_reconfigure default bounds
feature_extraction:
  x_min: 3.0
  x_max: 7.0
  y_min: -3.0
  y_max: 3.0
  z_min: -1.7
  z_max: 0.0
  k: 50
  z: 1
  voxel_res: 0.4

# Properties of chessboard calibration target
chessboard:
  pattern_size:
    height: 7
    width: 5
  square_length: 105
  board_dimension:
    width: 700.5
    height: 1000
  translation_error:
    x: 2
    y: 2
